### ISSUE TITLE
Prepare for first public release

### ISSUE THREAD

**Issue by robbiemccorkell** [redbadger/pride-london-app#71]
We should start thinking about the things we still need (that aren't currently represented in the backlog) before we can do our first release.

Some things I can think of are:
- [ ] Decide on and set up CI prod release mechanism
- [x] Enable app signing in play store
- [ ] Figure out a mechanism for transferring our test content model to the production space
- [ ] Allow differing environment variables for each CI build stage. We will need to pass the app the API key for the production contentful space. Maybe the contentful test space for alpha builds and prod space for beta/prod builds?
- [ ] Create a production API key for Google maps with appropriate permissions.
- [ ] Add Microsoft Codepush library and workflow for over the air builds?
- [ ] Submit a stable build for iOS app store review without publishing it, to test how long it takes and see if there are any issues we need to sort out.
- [ ] Decide on app icons and splash screen
- [ ] Limit distribution to sensible devices (no tablets)
- [ ] Automate capturing required screenshots using Fastlane (part of prod release mecahnism)

Can anyone think of more?

**Comment by frigus02** [comment]
What kind of information do we need for the App Store or Play Store?

* For example I'm pretty sure that every app on the Play Store needs a public privacy policy hosted somewhere. Is it okay to use the one on https://prideinlondon.org/terms?

**Comment by robbiemccorkell** [comment]
Quite possibly, although if there's an app set up there already that we'll be updating, maybe this is already done

**Comment by markholland** [comment]
Couldn't see App icons in Trello.

 - If there's a new icon we need to do it.
 - If we want to use the existing one we need access to a high res version.

**Comment by robbiemccorkell** [comment]
Added app icons to the list above.

**Comment by robbiemccorkell** [comment]
I've been having another look at appcenter.me and it looks like you can push your own binaries to the services without having to use their build pipeline with the `appcenter distribute release` command in their CLI.

Appcenter can handle the distribution to stores, but also crash reporting, analytics, push notifications, and codepush releases in one place. I'm not sure how good it is at all these things compared to competitors, but I'll try and upload a build soon and try them out.

**Comment by markholland** [comment]
Some launch checklists to go through as content will likely come from Pride org although I presume most won't change from whatever the live version has.

Android: https://developer.android.com/distribute/best-practices/launch/launch-checklist.html
iOS: https://blog.oursky.com/2017/03/09/checklist-ios-app-store-itunes-connect/

**Comment by markholland** [comment]
This is the privacy policy currently set in both stores http://prideinlondon.org/about-us/privacy

Going to that link just shows the homepage, no redirect.

**Comment by markholland** [comment]
We did it
